src/octoprint/util/gcodeInterpreter.py

```diff
@@ -271,6 +280,8 @@ def _load(self, gcodeFile, printer_profile, throttle=None):
 	def _parseCuraProfileString(self, comment, prefix):
 		return {key: value for (key, value) in map(lambda x: x.split("=", 1), zlib.decompress(base64.b64decode(comment[len(prefix):])).split("\b"))}
 
+	def getWarning(self):
```

Comment:
This is not Java, we don't need getters for everything :) (Yes, I know, I've done that myself, I'm trying to get rid of such things while refactoring)